Specs/MiddlewareLib/0.8.0/MiddlewareLib.podspec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "MiddlewareLib",
+  "version": "0.8.0",
+  "summary": "Library that is the middleware between NPC´s Apps and NPC´s services",
+  "homepage": "https://github.com/indigotech/br-npc-middleware-lib-java/",
+  "license": {
+    "type": "Comercial"
```

Comment:
@mmassaki copied from here https://github.com/indigotech/Specs/blob/master/Specs/TaqtileSDK/0.8.2/TaqtileSDK.podspec.json, think we should add issue to change this on TQSDK spec too?